crdppf/util/get_feature_functions.py

```diff
@@ -40,18 +40,17 @@ def get_features_function(parcelGeom, params):
                 if geometryType == 'ST_Polygon' or geometryType == 'ST_MultiPolygon':
                     intersectionMeasure = DBSession.scalar(feature.geom.intersection(parcelGeom).area())
                     if intersectionMeasure >= 1:
-                        intersectionMeasureTxt = str(math.ceil(intersectionMeasure*10)/10) + ' [m2]'
+                        intersectionMeasureTxt = ' : '+ str(int(intersectionMeasure)) + ' [m2]'
```

Comment:
unfortunately not only. But I'll give it a try anyway in case fpdf accepts it also. :)